### DISCUSSION TITLE
PHP wrapper?

### DISCUSSION THREAD

**Discussion by aolko** [webui-dev/webui#350] [Q&A]
Can somebody make a php wrapper?

**Comment by AlbertShown** [comment]
PHP (or any other back-end language) should have those features to be able to use WebUI:

- Load a dynamic library `.dll` `.so` `.dynlib`
- Have FFI feature to use dynamic library's APIs
- Stay running async in background until `webui_wait()` breaks

If PHP has this requirement, then yes, it's possible.

**Reply by KingBes** [reply]
Yes. But I can't use webview, and I don't know why. [php-webui](https://github.com/KingBes/php-webui-composer)

**Reply by AlbertShown** [reply]
In Windows you need to put Microsoft WebView Loader DLL in the same directory.

**Reply by KingBes** [reply]
What about Linux and Mac environments？

**Reply by AlbertShown** [reply]
macOS always has the necessary webview runtimes installed, so all good.
Linux I guess you should install the GTK webview v3 to be able to use webview...
however, main focus of webui is web browsers.
